UefiTestingPkg/AuditTests/PagingAudit/UEFI/DxePagingAuditTestApp.inf

```diff
@@ -18,13 +18,20 @@
 
 [Sources]
   Dxe/App/DxePagingAuditTestApp.c
+  Dxe/App/DxePagingAuditTestApp.h
   PagingAuditCommon.c
   PagingAuditCommon.h
 
-
 [Sources.X64]
   X64/PagingAuditProcessor.c
+  Dxe/App/x86/DxePagingAuditTestsx86.c

```

Comment:
nit: naming, is it more consistent to use x64? Same I guess with aarch64.

Reply:
Renamed